django_celery_beat/models.py

```diff
@@ -348,7 +385,10 @@ class PeriodicTask(models.Model):
         help_text=_('Solar Schedule to run the task on.  '
                     'Set only one schedule type, leave the others null.'),
     )
-
+    clocked = models.ForeignKey(
+        ClockedSchedule, on_delete=models.CASCADE, null=True, blank=True,
+        verbose_name=_('clocked'), help_text=_('Use a clocked schedule')
```

Comment:
Uppercase verbose_name as that's used for the label in any UI (e.g. siteadmin), and please make the help_text match the solar/crontab/interval fields since this is another schedule type along with those.

Reply:
Done